### ISSUE TITLE
[Bug] AutomationId not usable for automated UI testing

### ISSUE THREAD

**Issue by GuillaumeMorinQc** [dotnet/maui#3260]
### Description
AutomationId property in xaml doesn't generate content-desc property in android apk.
 This is used by ui automated tests tools. The current work around 
### Steps to Reproduce

1. Create a new maui app or use Hello-maui example app
2. add an AutomationId on the button or on labels
3. Generate an android .apk
4. Inspect the apk with a layout inspector (ex.:android studio, appium inspector)

### Expected Behavior
The element with AutomationId should have a content-desc property with the AutomationId value
### Actual Behavior
The element doesn't have the content-desc
### Basic Information

- Version with issue: maui preview 9

- Platform Target Frameworks: 
  - Android: 9.0, 10.0, 11.0

### Workaround

Setting the SemanticProperties.Description instead generate the correct property in the apk.
